docs/changes/cosmology/12375.api.rst

```diff
@@ -1,3 +1,3 @@
 ``default_cosmology.get_cosmology_from_string`` is deprecated and will be
 removed in two minor versions.
-Use ``default_cosmology.get(<str>)`` instead.
+Use ``from astropy.cosmology import <str>`` instead.
```

Comment:
These are not really equivalent as the import cannot be done programmatically. Maybe explicitly say
```
from astropy import cosmology
getattr(cosmology, <str>)
```
Or is that too obvious?

Reply:
SGTM. It's just to note that the ability to get the realizations isn't completely gone.